src/System.Runtime.Numerics/tests/BigInteger/cast_from.cs

```diff
@@ -564,6 +575,12 @@ public static void RunDoubleExplicitCastFromBigIntegerTests()
         }
 
         [Fact]
```

Comment:
Yes, please go ahead and squash this down to one commit. You shouldn't need to create a new PR, just (force) push to the branch you are using now and GitHub will pick up the changes.